integration-tests/src/test/java/oracle/kubernetes/operator/utils/TestUtils.java

```diff
@@ -1200,23 +1215,38 @@ public static void buildWdtZip(
   }
 
   public static void createDirUnderDomainPV(String dirPath) throws Exception {
-    dirPath = dirPath.replace(BaseTest.getPvRoot(), "/sharedparent/");
-    String crdCmd =
-        BaseTest.getProjectRoot()
-        + "/src/integration-tests/bash/krun.sh -m " + BaseTest.getPvRoot() + ":/sharedparent -c 'mkdir -m 777 -p "
-        + dirPath
-        + "'";
-    
-    ExecResult result = ExecCommand.exec(crdCmd);
-    if (result.exitValue() != 0) {
-      throw new RuntimeException(
+    if (BaseTest.OPENSHIFT) {
+      String crdCmd = "mkdir -m 777 -p " +  dirPath;
+  
+      ExecResult result = ExecCommand.exec(crdCmd);
+      if (result.exitValue() != 0) {
+        throw new RuntimeException(
           "FAILURE: command to create domain scripts directory "
               + crdCmd
               + " failed, returned "
               + result.stdout()
               + result.stderr());
+      }
+      logger.info("command result " + result.stdout().trim());
+    } else {
+      dirPath = dirPath.replace(BaseTest.getPvRoot(), "/sharedparent/");
+      String crdCmd =
+          BaseTest.getProjectRoot()
+          + "/src/integration-tests/bash/krun.sh -m " + BaseTest.getPvRoot() + ":/sharedparent -c 'mkdir -m 777 -p "
+          + dirPath
+          + "'";
+      
+      ExecResult result = ExecCommand.exec(crdCmd);
```

Comment:
as you are changing this code, please do the same here ...replace these lines with TestUtils.exec(str, true)